cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -36,10 +36,17 @@
     public static final String STRIMZI_NAMESPACE = "STRIMZI_NAMESPACE";
     public static final String STRIMZI_FULL_RECONCILIATION_INTERVAL_MS = "STRIMZI_FULL_RECONCILIATION_INTERVAL_MS";
     public static final String STRIMZI_OPERATION_TIMEOUT_MS = "STRIMZI_OPERATION_TIMEOUT_MS";
-    public static final String STRIMZI_CREATE_CLUSTER_ROLES = "STRIMZI_CREATE_CLUSTER_ROLES";
     public static final String STRIMZI_IMAGE_PULL_POLICY = "STRIMZI_IMAGE_PULL_POLICY";
     public static final String STRIMZI_IMAGE_PULL_SECRETS = "STRIMZI_IMAGE_PULL_SECRETS";
 
+    // Feature Flags
+    public static final String STRIMZI_ROLES_ONLY = "STRIMZI_ROLES_ONLY";
+    public static final boolean DEFAULT_STRIMZI_ROLES_ONLY = false;
```

Comment:
Is it really a boolean or should it be `role_mode` being `cluster` or `namespace` for now, but leaving the door open to some other alternative in future?

Reply:
Makes sense, will switch to this.